Rakefile

```diff
@@ -1,18 +1,18 @@
 require "bundler/gem_tasks"
 require 'rspec/core/rake_task'
+require 'rototiller'
 
 task :default => :test
 
 desc "Run spec tests"
 RSpec::Core::RakeTask.new(:test) do |t|
   t.rspec_opts = ['--color']
-  t.pattern = 'spec/'
+  t.pattern = ENV['SPEC_PATTERN']
```

Comment:
i *seeee* 
yeah, in this one we need a separate task to check because we're not using a rototiller task here, it's an rspec RakeTask, so we can't futz with the command using EnvVars.  
but i think in our acceptance task we could?  maybe having them in separate tasks is best practice?  

can you spell out these two cases in the README/docs?